handbook/product.md

```diff
@@ -388,6 +388,23 @@ Fleet's feature flag guidelines borrows from GitLab's ["When to use feature flag
 
 We track the capabilities of competitors and adjacent (or commonly integrated) products in this [Google Doc](https://docs.google.com/document/d/1Bqdui6oQthdv5XtD5l7EZVB-duNRcqVRg7NVA4lCXeI/edit) (private).
 
+## Product rituals
+
+The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.
+
+## Brand rituals
+
+The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.
+
```

Comment:
```suggestion
```
It looks like this was pasted from a different section of the handbook. I could be totally wrong here.